install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1786,6 +1786,10 @@ nodeIPAM:
   # -- Configure Node IPAM
   # ref: https://docs.cilium.io/en/stable/network/node-ipam/
   enabled: false
+lbIPAM:
```

Comment:
nit: LB is also an acronym, would the consistent thing to do be to make this `LBIPAM`?